app/services/TwoStepVerification.scala

```diff
@@ -39,37 +38,43 @@ trait TwoStepVerification {
 
   def twoStepVerificationThrottle: TwoStepVerificationThrottle
 
-  case class Biz2SVRule(name: String, conditions: List[Condition])
+  def twoStepVerificationRuleFactory = new TwoStepVerificationUserSegmentFactory {}
```

Comment:
TwoStepVerificationUserSegmentFactory doesn't need any overriding. That's why didn't bother to create an object for it. 
Yes the value could be a val instead of def.